fabric-rendering-fluids-v1/src/main/java/net/fabricmc/fabric/api/client/render/fluid/v1/SimpleFluidFogSettings.java

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
This is a simple implementation of the fog settings imterface.
IMO almost all modded fluids will use this, with fixed fog color and fixed parameters.